clang/tools/libclang/CXStoredDiagnostic.cpp

```diff
@@ -33,14 +33,14 @@ CXDiagnosticSeverity CXStoredDiagnostic::getSeverity() const {
     case DiagnosticsEngine::Error:   return CXDiagnostic_Error;
     case DiagnosticsEngine::Fatal:   return CXDiagnostic_Fatal;
   }
-  
+
```

Comment:
A lot of unrelated formatting changes (feel free to land as NFC and rebase on top of it if you want, but this keeps blames cleaner).